### PR TITLE
[Fix] Fixed organization dashboard UI issues

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
Fliplet/fliplet-studio#6712

## Description
Changed styling. Added small fixes.

## Screenshots/screencasts
https://streamable.com/6pgi82

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: src/components/tables/DataTable.vue [hunk 1]
```diff
@@ -75,6 +75,14 @@ export default {
       this.component.columns(colIndex)
         .search(event.target.value)
         .draw();
+    },
+    onKeydown: function(event) {
+      if (event.which === 65 && (event.ctrlKey || event.metaKey)) {
```
Comment (tonytlwu) [inline src/components/tables/DataTable.vue:80]: By this logic, does it mean that Cmd + C/X/V also don't work? If so, why is it that we need to be writing our own event listeners for the native browser features?

Comment (MaksymShokin) [reply src/components/tables/DataTable.vue:80]: Actually everything except ctrl + a worked fine before. Now everything works fine.

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: Organisation dashboard users
Verify: Dashboard features work as expected

File: src/components/tables/DataTable.vue (hunk 1 above)
Comment (tonytlwu) [reply src/components/tables/DataTable.vue:80]: @MaksymShokin Thanks. I've approved the work. Could you leave a comment to explain why Ctrl + A didn't work so I can understand?

**Review by squallstar — APPROVED** [review]

File: src/components/tables/DataTable.vue (hunk 1 above)
Comment (MaksymShokin) [reply src/components/tables/DataTable.vue:80]: @tonytlwu 
Actually I do know why  Ctrl + A did not work. Apparently, library has some issues with it. I did not find anything regarding it in the library docs. In the analytics Ctrl + A does not work also.

**Pull request merged**